ONSdigital/rm-survey-service
# Change BRES classifiers from RU_REF to FORM_TYPE for 2024

# What and why?

BRES 2024 will be using the FORM_TYPE classifier and not RU_REF. This db_migration updates those is the database.

# How to test

Following building your own dev namespace, `/deploy` this PR to your environment and ensure the changes in the `.up` delta 16 are applied. I'm unsure how the automated deployment of the `.down` backout SQL is applied, but you can manually apply the SQL it contains with PgAdmin or psql to see the configuration reverted.

Once delta 16 is applied and the records are updated, create a new BRES collection exercise and ensure you can upload a SURVEY_PERIOD_0001.xlsx SEFT in Rops (i.e. this is a standard form type filename and NOT a 49900000001.xlsx filename as has been the case in previous BRES collection exercises). Progress it through to Go Live and register a respondent, and download the file, then upload a response.
